WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuCoordinator.swift

```diff
@@ -60,10 +60,12 @@ final class HubMenuCoordinator: Coordinator {
             navigationController.viewControllers = [hubMenuController]
         }
 
-        if inactiveNotificationsSubscription == nil {
-            inactiveNotificationsSubscription = pushNotificationsManager.inactiveNotifications.sink { [weak self] in
-                self?.handleInactiveNotification($0)
-            }
+        if notificationsSubscription == nil {
+            notificationsSubscription = Publishers
+                .Merge(pushNotificationsManager.inactiveNotifications, pushNotificationsManager.foregroundNotificationsToView)
+                .sink { [weak self] in
+                    self?.handleInactiveNotification($0)
```

Comment:
Nit: I think this method should be renamed to `handleNotification` to be more generic now that we're handling both inactive and foreground notifications 😅

Reply:
yup good point! renamed in https://github.com/woocommerce/woocommerce-ios/pull/5857/commits/d674fecb4a73da528adb78a3be822d3e695153b3